src/test/scala/ch/epfl/bluebrain/nexus/tests/NewBaseSpec.scala

```diff
@@ -0,0 +1,214 @@
+package ch.epfl.bluebrain.nexus.tests
+
+import java.util.regex.Pattern.quote
+
+import akka.http.javadsl.model.headers.HttpCredentials
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.Authorization
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.http.scaladsl.unmarshalling.FromEntityUnmarshaller
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient
```

Comment:
we need to move away from this dependency as it will not be ported to the new repo